spec/factories/shopping_lists.rb

```diff
@@ -4,7 +4,7 @@
   factory :shopping_list do
     game
 
-    sequence(:title) { |n| "My List #{n}" }
+    sequence(:title) { |n| "Shopping List #{n}" }
```

Comment:
I didn't want the default title in the factory to interfere with the model's default title behaviour.